lib/Dialect/HW/HWAttributes.cpp

```diff
@@ -1084,3 +1084,31 @@ bool hw::isParametricType(mlir::Type t) {
       })
       .Default([](auto) { return false; });
 }
+
+WaiverAttr WaiverAttr::getWaiveXassign(::mlir::MLIRContext *context) {
+  return get(context, hw::WaiverKind::NoAssignX);
+}
+
+WaiverAttr WaiverAttr::getExcludeCoverage(::mlir::MLIRContext *context) {
+  return get(context, hw::WaiverKind::ExcludeCoverage);
+}
+
+std::pair<std::string, std::string> WaiverAttr::getWaiverComment() {
+  switch (getWaiverKind()) {
+  case hw::WaiverKind::NoAssignX:
+    return {"//spyglass disable_block NoAssignX-ML",
+            "//spyglass enable_block NoAssignX-ML"};
+  case hw::WaiverKind::ExcludeCoverage:
+    return {"//VCS coverage exclude_file", ""};
+  }
```

Comment:
Can we insert a whitespace?
```suggestion
    return {"// spyglass disable_block NoAssignX-ML",
            "// spyglass enable_block NoAssignX-ML"};
  case hw::WaiverKind::ExcludeCoverage:
    return {"// VCS coverage exclude_file", ""};
  }
```